CIPs/0019.md

```diff
@@ -0,0 +1,72 @@
+---
+cip: 0019
+title: JWT signer for DID-JWT
+author: @daoauth
+discussions-to: https://github.com/celo-org/celo-proposals/issues/73
+status: Draft
+type: 
+category: 
+created: 2020-09-09
+license: Apache 2.0
+---
+
+## Simple Summary
+JWT encryption is required for web and did support.
```

Comment:
Can you define the acronym for JWT before using it throughout the doc? It makes for easier reading.

Also what do you mean by "did support"?

Further more, can you explain why JWT is required? Is it for security, for speed, etc.?

Reply:
DID and JWT definitions have been updated.